weaver/converter/athenapdf/athenapdf.go

```diff
@@ -22,19 +22,30 @@ type AthenaPDF struct {
 	// an '-A' command-line flag to indicate aggressive content extraction
 	// (ideal for a clutter-free reading experience).
 	Aggressive bool
+	// NoPortrait will set the output PDF to be in landscape instead of in
+	// portrait orientation
+	NoPortrait bool
+	// Sets the page size for the PDF
+	PageSize string
 }
 
 // constructCMD returns a string array containing the AthenaPDF command to be
 // executed by Go's os/exec Output. It does this using a base command, and path
 // string.
 // It will set an additional '-A' flag if aggressive is set to true.
 // See athenapdf CLI for more information regarding the aggressive mode.
-func constructCMD(base string, path string, aggressive bool) []string {
+func constructCMD(base string, path string, aggressive bool, noPortrait bool, pageSize string) []string {
```

Comment:
Also IMHO this method should receive the AthenaPDF instance and take the params from there. Not sure why the struct was split out to separate params.